### PR TITLE
[DOCS] Reformats simple query string query

### PR DESCRIPTION
**Pull request by jrodewig**
Updates the `simple_query_string` query to use the new query format.

This creates separate sections for the example request, parameters, and notes.

This is part of #40977, an effort to standardize documentation for query types.

### Preview
http://elasticsearch_45343.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/query-dsl-simple-query-string-query.html

### CODE REVIEW

**Review by mayya-sharipova — APPROVED** [review]

**Comment by mayya-sharipova** [comment]
@jrodewig Thanks, nice reorganization. I have left a couple of small comments

File: docs/reference/query-dsl/simple-query-string-query.asciidoc [hunk 1]
```diff
@@ -24,72 +35,108 @@ GET /_search
...
-some analyzers will be not able to provide a meaningful results
-based just on the prefix of a term. Defaults to `false`.
+`analyze_wildcard`::
+(Optional, boolean) If `true`, the query attempts to analyze wildcard terms in
+the query string. Defaults to `false`.
 
-|`lenient` | If set to `true` will cause format based failures
-(like providing text to a numeric field) to be ignored.
+`analyzer`::
+(Optional, string) <<analysis,Analyzer>> used to convert unquoted text in the
```
[the remark this answers is not in the document]
Comment (jrodewig) [reply docs/reference/query-dsl/simple-query-string-query.asciidoc]: Great catch. This should have been removed in an earlier revision. Fixed with da556427527.

File: docs/reference/query-dsl/simple-query-string-query.asciidoc [hunk 2]
```diff
@@ -24,72 +35,108 @@ GET /_search
...
+--
+(Optional, array of strings) Array of fields you wish to search.
+
+This field accepts wildcard expressions. You also can boost relevance scores for
+matches to particular fields using a caret (`^`) notation. See
+<<simple-query-string-boost>> for examples.
+
+Defaults to the `index.query.default_field` index setting, which has a default
+value of `*`. The `*` value extracts all eligible fields in the mapping
+to term queries and filters the metadata fields. All extracted fields are then
```
[the remark this answers is not in the document]
Comment (jrodewig) [reply docs/reference/query-dsl/simple-query-string-query.asciidoc]: Fixed with 02236d4500e

File: docs/reference/query-dsl/simple-query-string-query.asciidoc [hunk 3]
```diff
@@ -4,10 +4,21 @@
 <titleabbrev>Simple query string</titleabbrev>
 ++++
 
-A query that uses the SimpleQueryParser to parse its context. Unlike the
-regular `query_string` query, the `simple_query_string` query will never
-throw an exception, and discards invalid parts of the query. Here is
-an example:
+Returns documents based on a provided query string, using a limited but
```
[the remark this answers is not in the document]
Comment (jrodewig) [reply docs/reference/query-dsl/simple-query-string-query.asciidoc]: Re-added with 4bcb3284237.

**Pull request merged**